backend/settings.py

```diff
@@ -16,26 +16,26 @@
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 from os.path import join, dirname, realpath
 
+import environ
+
+env = environ.Env()
+environ.Env.read_env()
+
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/2.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.environ.get(
```

Comment:
What is the advantage of `environ` over `os.environ` ?

Reply:
You mean, for `SECRET_KEY`?
`environ` uses a `.env` file
I don't know if you can achieve the same with built-in `os.environ`, I don't think so

Reply:
Ok 👍